### PR TITLE
Update UG and APP parameters

### PR DESCRIPTION
**Pull request by Originalidk**

### CODE REVIEW

**Review by wesho1107 — APPROVED** [review]
Good catches! LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1463,12 +1463,12 @@ Format: `createconsult d/DATE tt/TIME n/STUDENT_NAME…`
 #### 📝️Adding Students to a Consultation: `addtoconsult`
 You can add your student(s) into a consultation slot if you need to track students in your consultations.
 
-Format: `addtoconsult CONSULTATION_INDEX n/STUDENT_NAME…`
+Format: `addtoconsult CONSULTATION_INDEX [n/NAME]…`
```
Comment (raydenlim) [inline docs/UserGuide.md]: names optioal?

**Review by raydenlim — APPROVED** [review]
Other than 1 issue, LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (Originalidk) [reply docs/UserGuide.md]: Good catch, fixed!

**Pull request merged**